CheckersClient/src/main/java/cst/Checkers.java

```diff
@@ -1,39 +1,210 @@
-	private static final long serialVersionUID = 1L;
+   private static final long serialVersionUID = 1L;
+   static int playerOne1x = 0;
+   static int playerOne1y = 0;
+   static int playerOne2x = 0;
+   static int playerOne2y = 0;
+   static int playerOne3x = 0;
+   static int playerOne3y = 0;
+   static int playerOne4x = 0;
+   static int playerOne4y = 0;
+   static int playerOne5x = 0;
+   static int playerOne5y = 0;
+   static int playerOne6x = 0;
+   static int playerOne6y = 0;
+   static int playerOne7x = 0;
+   static int playerOne7y = 0;
+   static int playerOne8x = 0;
+   static int playerOne8y = 0;
+   static int playerOne9x = 0;
+   static int playerOne9y = 0;
+   static int playerOne10x = 0;
+   static int playerOne10y = 0;
+   static int playerOne11x = 0;
+   static int playerOne11y = 0;
+   static int playerOne12x = 0;
+   static int playerOne12y = 0;
+   static int playerTwo1x = 0;
+   static int playerTwo1y = 0;
+   static int playerTwo2x = 0;
+   static int playerTwo2y = 0;
+   static int playerTwo3x = 0;
+   static int playerTwo3y = 0;
+   static int playerTwo4x = 0;
+   static int playerTwo4y = 0;
+   static int playerTwo5x = 0;
+   static int playerTwo5y = 0;
+   static int playerTwo6x = 0;
+   static int playerTwo6y = 0;
+   static int playerTwo7x = 0;
+   static int playerTwo7y = 0;
+   static int playerTwo8x = 0;
+   static int playerTwo8y = 0;
+   static int playerTwo9x = 0;
+   static int playerTwo9y = 0;
+   static int playerTwo10x = 0;
+   static int playerTwo10y = 0;
+   static int playerTwo11x = 0;
+   static int playerTwo11y = 0;
+   static int playerTwo12x = 0;
+   static int playerTwo12y = 0;
+   static int newGame = 0;
 
     /**
      * Default constructor. 
      */
-    public Checkers() {
-        // TODO Auto-generated constructor stub
+    public Checkers() 
+    {
+        super();
     }
 
-	/**
-	 * @see HttpServlet#doGet(HttpServletRequest request, HttpServletResponse response)
-	 */
-	protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
-	   request.getRequestDispatcher("checkers.jsp").forward(request, response);
-		// TODO Auto-generated method stub
-		// response.getWriter().append("Served at: ").append(request.getContextPath());
-	}
-
-	/**
-	 * @see HttpServlet#doPost(HttpServletRequest request, HttpServletResponse response)
-	 */
-	protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
-		// TODO Auto-generated method stub
-		doGet(request, response);
-	}
+   /**
+    * @see HttpServlet#doGet(HttpServletRequest request, HttpServletResponse response)
+    */
+   protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+      request.getRequestDispatcher("checkers.jsp").forward(request, response);
+      response.getWriter().append("Served at: ").append(request.getContextPath());
```

Comment:
We can probably remove this line. I don't think it will be hit because it's being forwarded to the checkers page first.